CODEOWNERS

```
@@ -0,0 +1,9 @@
+# This file provides an overview of code owners in this repository.
+
+# Each line is a file pattern followed by one or more owners.
+# The last matching pattern has the most precedence.
+# For more details, read the following article on GitHub: https://help.github.com/articles/about-codeowners/.
+
+# The default owners are automatically added as reviewers when you open a pull request unless different owners are specified in the file.
+
+* @smoya @jonaslagoni @BOLT04 @magicmatatjahu @github-actions[bot]
```

I tried to bribe @jonaslagoni with some cinema tickets but didn't work, even though [The Matrix Resurrections](https://www.imdb.com/title/tt10838180/) is almost here... So I think we should remove it from the list.
```suggestion
* @smoya @BOLT04 @magicmatatjahu @github-actions[bot]
```